renderdoc/common/common.h

```diff
@@ -42,7 +42,12 @@
 #include <signal.h>
 
 #define __PRETTY_FUNCTION_SIGNATURE__ __PRETTY_FUNCTION__
+#if defined(RENDERDOC_PLATFORM_SWITCH)
```

Comment:
As I mentioned in the other PR, these checks should be using `RDOC_SWITCH`. `RENDERDOC_PLATFORM_SWITCH` should only be used in `globalconfig.h` to be able to define `RDOC_SWITCH`, or in `apidefs.h` since it's a public header and can't include `globalconfig.h`. All other uses should use `RDOC_SWITCH`.

Reply:
Sorry, I misread your previous comment and thought you meant to use RENDERDOC_PLATFORM_SWITCH  for the common files.  I will fix this.